packages/venia-ui/lib/components/ProductSort/productSort.js

```diff
@@ -15,6 +15,43 @@ const ProductSort = props => {
     const { availableSortMethods, sortProps } = props;
     const [currentSort, setSort] = sortProps;
     const { elementRef, expanded, setExpanded } = useDropdown();
+    const { formatMessage } = useIntl();
+
+    const orderSortingList = useCallback(
+        list => {
+            return list.sort((a, b) => {
+                const aText = formatMessage({
+                    id: a.id,
+                    defaultMessage: a.text
+                });
+                const bText = formatMessage({
+                    id: b.id,
+                    defaultMessage: b.text
+                });
+
+                return aText.localeCompare(bText, undefined, {
```

Comment:
If `formatMessage` returns the localized string, why do a `localeCompare` instead of a regular alphabetical order with `sort()`?